pkg/proxy/iptables/proxier.go

```diff
@@ -693,36 +683,46 @@ func portProtoHash(servicePortName string, protocol string) string {
 	return encoded[:16]
 }
 
-// servicePortChainName takes the ServicePortName for a service and
-// returns the associated iptables chain.  This is computed by hashing (sha256)
-// then encoding to base32 and truncating with the prefix "KUBE-SVC-".
-func servicePortChainName(servicePortName string, protocol string) utiliptables.Chain {
+// serviceChainName returns the name of the KUBE-SVC-XXXX chain for a service, which is the
+// main iptables chain for that service, used for dispatching to endpoints when using `Cluster`
+// traffic policy.
+func serviceChainName(servicePortName string, protocol string) utiliptables.Chain {
 	return utiliptables.Chain("KUBE-SVC-" + portProtoHash(servicePortName, protocol))
 }
 
-// serviceFirewallChainName takes the ServicePortName for a service and
-// returns the associated iptables chain.  This is computed by hashing (sha256)
-// then encoding to base32 and truncating with the prefix "KUBE-FW-".
+// serviceLocalChainName returns the name of the KUBE-SVL-XXXX chain for a service, which
+// handles dispatching to local endpoints when using `Local` traffic policy. This chain only
+// exists if the service has `Local` internal or external traffic policy.
+func serviceLocalChainName(servicePortName string, protocol string) utiliptables.Chain {
+	return utiliptables.Chain("KUBE-SVL-" + portProtoHash(servicePortName, protocol))
```

Comment:
I'm totally happy to bikeshed the names...

I avoided changing KUBE-SVC and KUBE-XLB to keep some consistency/familiarity, but given that KUBE-SVC eventually becomes "the chain used for `Cluster` traffic policy` rather than `the primary service chain`, I thought it would be somewhat mnemonic to have SVC = Cluster / SVL = Local...

Reply:
Good call, "C for Cluster" + "L for Local" is actually pretty clever, works for me.